monitoring/uss_qualifier/scenarios/faa/uft/message_signing_start.md

```diff
@@ -0,0 +1,27 @@
+# Start message signing test scenario
+
+This test scenario instructs a mock USS to begin capturing message signing data.
+
+## Resources
+
+### mock_uss
```

Comment:
I am not aware of the specifics of UFT signing but as I understand it, there should be at some point a UFT Service mocked by mock_uss, perhaps naming the resource accordingly would be better than using `mock_uss` ? (eg UFT service)

Reply:
The way it sounds like message signing testing will be implemented is that InterUSS's mock_uss (the one in `monitoring`) will acquire an optional capability to 1) validate incoming signed messages (both requests of incoming calls and responses to outgoing calls) and 2) log message signing performance.  Like we currently have `ridsp`, `riddp`, `scdsc`, and `geoawareness` capability sets in mock_uss, this future PR would add `messagesigning` (or similar) which would have endpoints supporting instruction to start logging message signing performance and to finalize message signing logging.  None of the USSs under test would interact with the `messagesigning` endpoints -- instead, they would interact with other endpoints (specifically, the ASTM SCD endpoints from `scdsc`) which would use and log message signing while the `messagesigning` logging capability was active.  So, there wouldn't be a separate UFT service exactly -- instead, this is just expecting mock_uss to acquire additional controllable behaviors.  Basically, this would be similar to the `routes_behavior.py` routes for `ridsp`, only not specific to just one service.

Reply:
Understood, sounds good to me. thanks